resolwe_bio/processes/import_data/seq_reads.yml

```diff
@@ -218,3 +218,218 @@
       re-save-file-list fastqc_archive2 "${RW_FASTQC[@]}"
       re-save-list fastqc_url "${FW_FASTQC_URL[@]}"
       re-save-list fastqc_url2 "${RW_FASTQC_URL[@]}"
+
+- slug: files-to-fastq-single
+  name: Convert files to reads (single-end)
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: resolwebio/rnaseq:2.1.0
```

Comment:
Use `resolwebio/rnaseq:2.1.1` Docker image which has `Trimmomatic` installed. The same goes for the paired-end process.

Reply:
Fixed.